zboxcore/sdk/sdk.go

```diff
@@ -871,7 +877,7 @@ func CreateAllocationForOwner(owner, ownerpublickey string,
 		"owner_id":                      owner,
 		"owner_public_key":              ownerpublickey,
 		"expiration_date":               expiry,
-		"preferred_blobbers":            preferredBlobbers,
+		"preferred_blobbers":            allocationBlobbers,
```

Comment:
Keep this as `preferredBlobbers`, `allocationBlobbers` can be the `preferredBlobbers` concatenated with the ids returned from `get_alloc_blobbers`

Reply:
This is going to be removed once 684 gets in, prefferedBlobbers would be made Blobbers & the list of blobbers passed from zbox will be the prefferedBlobbers & then the blobbers fetched from eventsDB endpoint would be the desired blobbers.
Preffered + desired = dataParity + shardParity, this is the number of blobbers which we are gonna send to the new allocation request.